source/index.html

```diff
@@ -44,170 +45,199 @@
+                </h3>
+                <p class="advantage__number advantage__number--light">
+                  —&nbsp;№2&nbsp;—
+                </p>
+                <p class="advantage__description advantage__description--light">
+                  Да, по нему можно<br class="advantage__description-br-tablet"> пройти!<br class="advantage__description-br-desktop"> Если вы<br class="advantage__description-br-tablet"> осмелитесь, конечно
+                </p>
+              </div>
+              <img
+                class="advantage__image advantage__image--reverse"
+                src="img/index/advantage/photo-bridge.jpg"
+                width="520"
+                height="260"
+                alt="Фотография моста дьявола."
+              />
+            </li>
+            <li class="advantage__item advantage__item--gray-theme">
+              <h3 class="advantage__title">Небольшая<br class="advantage__description-br-tablet"><br class="advantage__description-br-desktop"> площадь</h3>
+              <p class="advantage__number">—&nbsp;№3&nbsp;—</p>
+              <p class="advantage__description">
+                Вce интересные места <br class="advantage__description-br-tablet"> находятся очень<br class="advantage__description-br-desktop"> близко
               </p>
-            </div>
-            <img
-              class="advantage__image advantage__image--reverse"
-              src="img/advantage-bridge.jpg"
-              width="520"
-              height="260"
-              alt="Фотография моста дьявола."
-            />
-          </li>
-          <li class="advantage__item advantage__item--gray-theme">
-            <h3 class="advantage__title">Небольшая площадь</h3>
-            <p class="advantage__number">—&nbsp;№3&nbsp;—</p>
-            <p class="advantage__description">
-              Вce интересные места находятся очень близко
-            </p>
-          </li>
-          <li class="advantage__item advantage__item--gray-theme">
-            <h3 class="advantage__title">Красивая дорога</h3>
-            <p class="advantage__number">—&nbsp;№4&nbsp;—</p>
-            <p class="advantage__description">
-              Ехать в Седону из Лас-Вегаса совсем не скучно!
-            </p>
-          </li>
-          <li class="advantage__item advantage__item--gray-theme">
-            <h3 class="advantage__title">Мало туристов</h3>
-            <p class="advantage__number">—&nbsp;№5&nbsp;—</p>
-            <p class="advantage__description">
-              Большенство едет в Гранд Каньон и толпится там
-            </p>
-          </li>
-        </ul>
+            </li>
+            <li class="advantage__item advantage__item--gray-theme">
+              <h3 class="advantage__title advantage__title-red-road">Красивая<br class="advantage__description-br-tablet"><br class="advantage__description-br-desktop"> дорога</h3>
```

Comment:
это не модификатор. у модификатора 2 черточки
```suggestion
              <h3 class="advantage__title advantage__title--red-road">Красивая<br class="advantage__description-br-tablet"><br class="advantage__description-br-desktop"> дорога</h3>
```